termine/announce.go

```diff
@@ -31,18 +33,7 @@ func init() {
 	cmdAnnounce.Run = RunAnnounce
 }
 
-func isStammtisch(date time.Time) (stammt bool, err error) {
-	err = db.QueryRow("SELECT stammtisch FROM termine WHERE date = $1", date).Scan(&stammt)
-	return
-}
-
-func announceStammtisch(date time.Time) {
-	loc, err := getLocation(date)
-	if err != nil {
-		log.Println("Kann Location nicht auslesen:", err)
-		return
-	}
-
+func announceStammtisch(t *data.Termin) error {
```

Comment:
cf. [variable name style](https://github.com/golang/go/wiki/CodeReviewComments#variable-names) and [named result parameters](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters) (even though this is about a parameter, but the same reasoning applies). In short: a) variable/parameter name lengths should be inversely correlated to the distance between declaration and use and b) the signature of a function should be readable and not stutter (`termin *data.Termin` would be an example of stuttering).

Your personal opinion on this particular styleguide item might differ (mine doesn't), but in any case I'd prefer to go with the clear mandate of the go culture here.